### DISCUSSION TITLE
Is it possible to get output from the CLI?

### DISCUSSION THREAD

**Discussion by MitchelPaulin** [hoppscotch/hoppscotch#2936] [Q&A]
I would like to use the CLI and dump all the request responses to files to use for testing in different contexts, is this currently possible today?

**Comment by jamesgeorge007** [comment]
Hi, the CLI doesn't currently support directly dumping all request responses to files for reuse in different testing contexts. On another note, it now offers the ability to generate JUnit reports for collection runs, you can find more information [here](https://docs.hoppscotch.io/documentation/clients/cli/overview#generate-junit-report-for-collection-runs).
